XIPs/xip-0-purpose-process.md

```diff
@@ -38,13 +38,21 @@ An XIP must meet certain minimum criteria. It must be a clear and complete descr
 
 ## XIP Workflow
 
-## Getting started
+### Get started with an idea
 
 Before advancing your idea through the XIP process into the Draft stage, it’s crucial to gather feedback from the community. This will help you vet your idea and assess its originality, potentially saving you time and effort. Engaging with the XMTP community will also help you gauge overall interest in your idea.
 
-Prototyping your proposal may also help to de-risk it and provide a reference implementation to anchor discussions around.
+**To get started with your idea**:
 
-To get started, you can **submit your idea as a new topic in the [XIP Ideas category in XMTP's GitHub Discussions](https://github.com/orgs/xmtp/discussions/categories/xip-ideas)**.
+1. Submit your idea as a new topic in the [Ideas & Improvements category](https://community.xmtp.org/c/development/ideas/54) in the XMTP Community Forums.
+
+1. If you've built a prototype for your idea, provide access to it in your topic. A prototype can help de-risk your idea and give people something to anchor discussions around.
+
+1. Be sure to include the date by which you want people to provide feedback. A good standard is giving people ~2 weeks, including 2 weekends.
+
+1. Share your idea with the community to get feedback. For example, beyond posting to the XMTP Community Forums, you might want to share a link to your idea topic with your networks on other platforms.
```

Comment:
I am curious about how XMTP Editors and XMTP Core Developers can help the idea author amplify their idea, if they want help. For example, with the idea author's permission, an XMTP Core Developer with a Twitter following, might share it to get more people interested? Is that helpful or appropriate?

Reply:
I think so! More feedback the better for this stage.

Reply:
After processing for a lil bit - I think the amplification action is permissionless - but whether employees of XMTP Labs should amplify is a question of credible neutrality that we'll need to discuss. Also, if an idea is amplified using the XMTP protocol X.com account - I believe it must amplify all or none to be credibly neutral. More to dig into here. But I think the doc as-is works.

Reply:
Makes total sense!